.github/workflows/on_pull_request.yml

```diff
@@ -57,21 +57,27 @@ jobs:
       - name: Run gradle unit tests
         uses: ./config/actions/gradle-check
         with:
+          access-token: ${{ secrets.PACKAGES_PAT }}
+          username: ${{ secrets.PACKAGES_USER }}
           version-code: ${{ steps.version-code.outputs.version-code }}
           version-name: ${{ steps.version-name.outputs.version-name }}
 
       - name: Run gradle instrumentation tests
         uses: ./config/actions/gradle-connected-test
         with:
+          access-token: ${{ secrets.PACKAGES_PAT }}
           architecture: 'arm64-v8a'
           device-profile: 'pixel_xl'
+          username: ${{ secrets.PACKAGES_USER }}
```

Comment:
Could move this up next to access-token so it's consistent with other steps?

Reply:
Applies to other steps in this file as well

Reply:
this is already consistent as it's alphabetised because alphabetising is next to godliness

Reply:
you are right!